cookbooks/fb_iptables/recipes/packages.rb

```diff
@@ -19,14 +19,20 @@
 # limitations under the License.
 #
 
-# c10s is opted out until EPEL 10 becomes available
-if (node.centos? && !(node.centos7? || node.centos8? || node.centos10?)) || node.fedora?
-  packages = ['iptables-legacy']
```

Comment:
So now iptables-legacy is not required in any way - even on C9?

Reply:
Correct. It's a strict subset of iptables-nft. And the overlapping files have the same checksum.

Reply:
note that we need to switch our internal deployment away from this first - iptables-legacy is not needed if you're on the stock kernel (that has CONFIG_NFT_COMPAT enabled, so it can still delegate to the old XTABLES backend) but is needed still at Meta

Reply:
also: on c9 and above you need iptables-nft, likewise on Fedora, but on EL8 - which is still supported (RHEL 8, Alma 8, Rocky, Oracle etc) - though idk if we want to support it - the package is still iptables